### PR TITLE
Feature: connect sentry to the app

### PR DESCRIPTION
**Pull request by akhlopiachyi**

### CODE REVIEW

File: public/config/env.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-        authUrl: 'http://localhost:9002/api/v2/barong',
-        tradeUrl: 'http://localhost:9002/api/v2/peatio',
-        applogicUrl: 'http://localhost:9002/api/v2/applogic',
-        rangerUrl: 'ws://localhost:9003/api/v2/ranger',
-        finexUrl: '',
+        authUrl: 'https://dev.yellow.openware.work/api/v2/barong',
+        tradeUrl: 'https://dev.yellow.openware.work/api/v2/peatio',
+        applogicUrl: 'https://dev.yellow.openware.work/api/v2/applogic',
+        rangerUrl: 'wss://dev.yellow.openware.work/api/v2/ranger',
+        finexUrl: 'https://dev.yellow.openware.work/api/v2/finex',
```
Comment (oyershov) [inline public/config/env.js:7]: please, undo these changes

File: src/containers/Errors/index.tsx [hunk 2]
```diff
@@ -1,12 +1,44 @@
+import * as Sentry from '@sentry/browser';
 import * as React from 'react';
 
-class ErrorWrapper extends React.Component {
+interface ErrorWrapperState {
+    eventId: any; // tslint:disable-line
```
Comment (oyershov) [inline src/containers/Errors/index.tsx:5]: Define type here instead of using ```any```

Comment (akhlopiachyi) [reply src/containers/Errors/index.tsx:5]: EventId can be presented in different types

File: public/config/env.js (hunk 1 above)
Comment (akhlopiachyi) [reply public/config/env.js:7]: Sure

File: src/containers/Errors/index.tsx (hunk 2 above)
Comment (oyershov) [inline src/containers/Errors/index.tsx:5]: please, define interface here

File: src/containers/Errors/index.tsx [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+            scope.setExtras(info);
+            const eventId = Sentry.captureException(error);
+            this.setState({eventId});
+        });
     }
 
     public render() {
+        if (this.state.hasError) {
+            return (
+                <button onClick={() => Sentry.showReportDialog({ eventId: this.state.eventId })}>Report feedback</button>
```
Comment (oyershov) [inline src/containers/Errors/index.tsx:38]: add translation here
